helm/cas-provision/templates/deployer/deployerRole.yaml

```
@@ -0,0 +1,181 @@
+# Permissions removed during migration:
+# {APIGroups:[""], Resources:["pods/log"], Verbs:["create" "delete" "deletecollection" "patch" "update"]}
+
+apiVersion: authorization.openshift.io/v1
+kind: Role
+metadata:
+  annotations:
+    openshift.io/description: A user who can deploy applications
+    openshift.io/reconcile-protect: "false"
+  name: {{ .Release.Namespace }}-deployer
```

Shouldn't the role name be the same in every namespace?
```suggestion
  name: {{ .Release.Name }}-deployer
```